cypress/e2e/HANKA.cy.js

```diff
@@ -0,0 +1,265 @@
+ /// <reference types="cypress" />
+
+
+//describe('Login Standard User', function () {
+it('Open website, type and login as Standard User', function() {    //user
+  cy.visit('/')
+
+  cy.get('[data-test="username"]')
+    .type('standard_user')
+    .should('have.value', 'standard_user')
+  cy.get('[data-test="password"]')
+    .type('secret_sauce')
+    .should('have.value', 'secret_sauce')
+  cy.get('[data-test="login-button"]')  
+    .click()  
+    .title().should('include', 'Swag Labs')
+})
+
+//describe('Login Blocked User', function () {
+it('Open website, type and login as Blocked User', function() {      //blocked user
+  cy.visit('/')
+
+  cy.get('[data-test="username"]')  //wpisuje login
+    .type('locked_out_user')
+    .should('have.value', 'locked_out_user')
+  cy.get('[data-test="password"]')     //wpisuje hasło
+    .type('secret_sauce')
+    .should('have.value', 'secret_sauce')
+  cy.get('[data-test="login-button"]')     // klikam przycisk logowania
+    .click()  
+  cy.get('[data-test="error"]')       //sprawdzam czy pojawił się oczekiwany błąd (napis error)
+    .should('contain', 'Epic sadface: Sorry, this user has been locked out.')
+}) 
+
+//describe('Login Problem User', function () {
+it('Open website, type and login as Problem User', function() {     //problem user
+  cy.visit('/')
+
+  cy.get('[data-test="username"]')
+    .type('problem_user')
+    .should('have.value', 'problem_user')
+  cy.get('[data-test="password"]')
+    .type('secret_sauce')
+    .should('have.value', 'secret_sauce')
+  cy.get('[data-test="login-button"]')  
+    .click()  
+    .title().should('include', 'Swag Labs')
+})
+
+//describe('Login Performance Glitch User', function () {
+it('Open website, type and login as Performance Glitch User', function() {      //performance glitch user
+  cy.visit('/')
+
+  cy.get('[data-test="username"]')
+    .type('performance_glitch_user')
+    .should('have.value', 'performance_glitch_user')
+  cy.get('[data-test="password"]')
+    .type('secret_sauce')
+    .should('have.value', 'secret_sauce')
+  cy.get('[data-test="login-button"]')  
+    .click()  
+    .title().should('include', 'Swag Labs')
+})
+
+//describe('Scroll page', function () { 
+it('Scroll page', function() {    //scroll page
+  cy.visit('/')
+
+  cy.get('[data-test="username"]')
+    .type('standard_user')
+  cy.get('[data-test="password"]')
+    .type('secret_sauce')
+  cy.get('[data-test="login-button"]')  
+    .click()  
+
+  cy.get('footer').scrollIntoView().should('be.visible')      //scroll do stopki
+
+  cy.get('head').scrollIntoView().title().should('include', 'Swag Labs')   //scroll do nagłówka
```

Comment:
`title()` method returns the value for the tab name not any element inside the page. I suggest asserting if `Products` is visible or if the logo is visible